src/Grpc/Interop/test/testassets/InteropWebsite/TestServiceImpl.cs

```diff
@@ -50,9 +47,10 @@ public override async Task StreamingOutputCall(StreamingOutputCallRequest reques
 
         foreach (var responseParam in request.ResponseParameters)
         {
+            // TODO(JamesNK): Remove nullable override after Grpc.Core.Api update
```

Comment:
Can we do this now after the 2.40.0 -> 2.43.0 updates in Version.props?

Reply:
No, it requires 2.45.0